### DISCUSSION TITLE
Fail2ban not automatic added logs files

### DISCUSSION THREAD

**Discussion by ahmedtanany** [fail2ban/fail2ban#3577] [Q&A] [answered]
Hi all,

I have fail2ban inside container on Debian Os. I used fail2ban to secure apache webserver which also inside the same container.

here is the filter inside jail.conf

[apache-log4j]
enabled  = true
filter   = apache-log4j
action = iptables-multiport[name=apache-log4j, port="http,81,https"]
logpath  = /apache/current/logs/access*.log
bantime  = 3600
maxretry = 1

but for some reason when a new day starts and apache creates a new access logs with the date like this access_offloaded.2023-09-20.log

fail2ban not added the new log file and i have to restart fail2ban to add the new log file

anyone can know hou can i make fail2ban automatically added a log files created dialy by apache without restarting fail2ban every day?

thanks

**Comment by sebres** [comment, marked as answer]
Yes this is known issue (e. g. #1379) unfortunately, see also #1407.
At the moment one has only few ways to achieve this:
- either to use persistent file name for logging (and create dated file names by log rotation only)
- or to create a hard link to some persistent name in logrotate `postrotate` event or by cron
- or to update them with `fail2ban-client set "$jail" dellogpath "$oldlogfile"; fail2ban-client set "$jail" addlogpath "$newlogfile";`
- or to use systemd journal instead of log-file
